weaver/core/relay/Cargo.toml

```diff
@@ -37,8 +37,8 @@ futures = { version = "0.3.27" }
 base64 = "0.20.0"
 reqwest = { version = "0.11.16", features = ["json"] }
 serde_json = "1.0.108"
-cacti_weaver_protos_rs = "2.0.0-rc.1"
-# cacti_weaver_protos_rs = "2.0.0-rc.1"
+cacti_weaver_protos_rs = "2.0.0-rc.2"
```

Comment:
Hi @petermetz Sorry I don't know how this gets modified, but can you fix this as follows:
```
# cacti_weaver_protos_rs = { path = "./protos-rs" }
cacti_weaver_protos_rs = "2.0.0-rc.3"
```
The order of line number is important, first one should point to local build, while 2nd line should point to the published module.

Reply:
@sandeepnRES There is no `2.0.0-rc.3` on crates.io, I set it to 2.0.0-rc.2 for now. Does that work?

Reply:
This is a release PR right? so this will release the new package `2.0.0-rc.3` for `protos-rs` like all other packages (where they also don't exist right now but will be available after the packages are published). And after they are published, it will work.

Reply:
@sandeepnRES Ohh, OK, now I understand! The build on the CI for this pull request will fail for now but I'll just mark those jobs as not required manually temporarily so that we can get over this hump. Thanks for the clarifications!